contrib/kube-prometheus/README.md

```diff
@@ -74,6 +74,34 @@ $ kubectl create -f manifests/ 2>/dev/null || true  # This command sometimes may
 $ kubectl delete -f manifests/ || true
 ```
 
+### Access the dashboards
+
+Prometheus, Grafana, and Alertmanager dashboards can be accessed after running the quickstart via the following commands:
```

Comment:
We should add a comment that in production environments it may make sense to additionally expose and secure these via ingress and or additional auth proxies.